FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/DefaultCommandItem.kt

```diff
@@ -0,0 +1,27 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+
+open class DefaultCommandItem(
```

Comment:
A question on the expectation for updating comment state after some command button is clicked - the default behaviour does not appear to allow for updating state, so I believe the expectation is that consumers will have a custom implementation of CommandItem which will update the internal state as needed, and that should be reflected in the UI after any event?

Reply:
On looking at this further, I do not see where the binding to update the UI state is, is there a missing call to update the CommandItem state whenever the data layer changes?

Reply:
Thanks, I confirmed that state now updates as you would expect